source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch, getPRRepositoryInfo, getRepositoryInfo, getUsername} from '../github-helpers';
+
+// Logic found on https://github.com/sindresorhus/refined-github/pull/3596#discussion_r511546864
+function getRemoteName(): string | undefined {
+	const author = getPRRepositoryInfo().owner;
+	if (author === getUsername()) {
+		return; // `origin`, don't add remote
+	}
+
+	if (author !== getRepositoryInfo().owner) {
+		return author;
+	}
+
+	if (select('[aria-label="Edit Pull Request title"]')) {
+		return; // It's a collaborator, it's likely to be `origin`
+	}
+
+	return 'upstream';
+}
+
+const connectionType: Record<string, string> = {
```

Comment:
```suggestion
const connectionType = {
```

Reply:
this is needed

Reply:
Indeed. You can skip it however if you change the `remoteType` type from `string` to `'HTTPS' | 'SSH'`

Alternatively, if you add `as const` after this object, *maybe* you can use `remoteType?: keysof connectionType`